src/formats/GRO.cpp

```diff
@@ -185,7 +185,7 @@ void GROFormat::write_next(const Frame& frame) {
 
         if (residue && residue->id()) {
             auto value = residue->id().value();
-            if (value <= 99999) {
+            if (value <= 99999 && value >= 0) {
```

Comment:
Does the GRO format not support negative id? If so, I would emit a warning if we find one of these.

Reply:
It's rather unclear. I cannot find any examples where the ID is negative and GROMACS treats residues as signed integers. Here is a case where it is expected for the residue ID to be an increasing whole number. The spec is rather unclear:

http://manual.gromacs.org/documentation/2018/user-guide/file-formats.html#gro